dev/README.md

```diff
@@ -578,4 +578,5 @@ Cheers,
 </p>
 </details>
 
-- Update the Announce page on [the wiki](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=62694597)
+- Update the Announcement page on
+[Airflow Site](https://github.com/apache/airflow-site/tree/master/landing-pages/site/content/en/announcements)
```

Comment:
Should be more explicit?
```suggestion
[Airflow Site](https://github.com/apache/airflow-site/blob/master/landing-pages/site/content/en/announcements/_index.md)
```

Reply:
I'd even say - why not link directly to https://airflow.apache.org/announcements/ ? 
When I updated it, I used the "suggest change on this page" link at the bottom and it worked perfectly :)

Reply:
cool will do that